lib/std/coff.zig

```diff
@@ -62,6 +62,21 @@ pub const MachineType = enum(u16) {
     Thumb = 0x1c2,
     /// MIPS little-endian WCE v2
     WCEMIPSV2 = 0x169,
+
+    pub fn toTargetCpuArch(machine_type: MachineType) std.target.Arch {
+        return switch (machine_type) {
+            .ARM => .arm,
+            .POWERPC => .powerpc,
+            .RISCV32 => .riscv3,
+            .Thumb => .thumb,
+            .Thumb => .thumbeb,
+            .I386 => .i386,
+            .ARM64 => .aarch64_32,
+            .ARM64 => .aarch64,
```

Comment:
There is a duplicate switch item here. Probably it should just be `.aarch64`?

Reply:
you are correct! I'm surprised that isn't a compiler error.

Reply:
Semantic Analysis should detect these duplicate switch branches and report an error, but I think this function is not referenced so it isn't analyzed.

Reply:
hmm - is there something I should be doing to force reference these then as part of the standard library? Either way I have checked a fix for this issue in now.

Reply:
There is https://github.com/ziglang/zig/blob/5fbae9cd6fe7a53d85ad6dd20cab75f7b835f0ad/lib/std/testing.zig#L508 but I don't know the conventions of the standard library well enough to say if its use is ok or if its not recommended.